core/src/components/radio/radio.scss

```diff
@@ -175,6 +181,10 @@ input {
   @include margin(0, $form-control-label-margin, 0, 0);
 }
 
+:host(.radio-label-placement-start.in-item) .label-text-wrapper {
+  @include margin($radio-item-label-margin-top, $form-control-label-margin, $radio-item-label-margin-bottom, 0);
```

Comment:
It might be easier to do this:

```scss
:host(.radio-label-placement-end) .label-text-wrapper {
  @include margin(null, null, null, $form-control-label-margin);
}

:host(.in-item:not(.legacy-radio)) .label-text-wrapper {
  @include margin($radio-item-label-margin-top, null, $radio-item-label-margin-bottom, null);
}
```

When you use `null` the margin values do not get set (rather than getting explicitly set to `0`). This approach lets you add the top and bottom margins without re-defining the start/end margins for each label placement.

Reply:
Ooh, good tip. I think I can actually remove the extra `.in-item` rules, since this lets me avoid overwriting the new `margin-top`/`margin-bottom` further up.